.vscode/launch.json

```diff
@@ -30,15 +30,15 @@
     {
       "name": "Launch Chrome",
       "request": "launch",
-      "type": "pwa-chrome",
```

Comment:
pwa-chrome is no longer supported on my version of VS Code, the type is now simply `chrome`